content/influxdb/v2.0/reference/cli/influx/v1/dbrp/update.md

```diff
@@ -0,0 +1,35 @@
+---
+title: influx v1 dbrp update
+description: >
+  The `influx v1 dbrp update` command updates a DBRP mapping in the InfluxDB 1.x compatibility API.
+menu:
+  influxdb_2_0_ref:
+    name: influx v1 dbrp set-active
+    parent: influx v1 dbrp
+weight: 101
+influxdb/v2.0/tags: [DBRP]
+---
+
+The `influx v1 dbrp update` command updates a DBRP mapping in the [InfluxDB 1.x compatibility API](/influxdb/v2.0/reference/api/influxdb-1x/).
+
+## Usage
+```
+influx v1 dbrp update [flags]
```

Comment:
is this missing the `--db` ?  seems like we can specify the `--rp` and it would be weird not to be able to also specify the `--db`.

Reply:
it is also a little odd that `-d` here is `--default` where elsewhere `-d` is for `--db`.  If this is the case, I think we document it, but file a CLI defect.  This inconsistency is undesirable.

In other commands, `--default` has no shortcut code.

Reply:
@timhallinflux 

- There's no `--db` in the help text. 

- default shortcut was an error--removed.

Reply:
@stuartcarnie -- can you cross check this for us ^^?

Reply:
I'm will remove the short-hand versions of the commands, like `-d` and `-r`, as they aren't enough of a benefit over `--db` and `--rp`, which are already short enough.

I will verify the semantics of the HTTP API to manipulate DB/RP resources (`/api/v2/dbrp`. I'm believe it currently doesn't allow you to rename a database for an existing DB/RP mapping. No reason it can't, but the behaviour was originally following the semantics of OSS v1.x, which doesn't allow you to rename a database. In v2's case, a user could simply create a new DB/RP mapping, with a new name, to an existing bucket and delete the old mapping.

Reply:
I can confirm the `database` cannot be updated. I recall this was by design, as there were implications when "renaming" a database.

I will prepare a PR to make some of these improvements and resolve some bugs in the `dbrp` commands.

Reply:
Issue influxdata/influxdb#20085 and PR influxdata/influxdb#20086 to address these issues and fix a number of bugs.

Reply:
Thanks @stuartcarnie and @timhallinflux -- I've updated the docs accordingly